common/config/presets/tari_sample.toml

```diff
@@ -42,3 +42,7 @@ monerod_password=""
 
 ## Will start mining only when base node has reported that it is bootstrapped
 #mine_on_tip_only=true
+
```

Comment:
This config file is intended to be a minimal set of options for runtime.
- Currently, none of the settings has any description, propose to remove the descriptions in this file.
- The minimal settings for the standalone miner to run should not be commented out.
- Maybe 8 threads are a bit too much for a default?

Reply:
I see, thanks, will remove settings. Actually mining node will start with no configuration (e.g. zero-configuration), by default it will use  `base_node.ridcully.grpc_base_node_address` to connect, so I guess can remove whole section from here. Added them here as it is clear how to connect to node running on another machine. 

I think @delta1 suggested that we need configuration in the sample config which makes sense for me, can you guys please point me to the right one I will place options there. Thanks

Regarding mining threads - as this is standalone mining node I suppose by default what would users want is to mine using all the possible CPU power, so it will use that if no options specified (e.g. it looks up number of logical CPU cores on user machine). `num_mining_threads` option allows to override that, e.g. if someone wants to mine just on one thread etc..

Reply:
Applied requested changes, keeping this comment open as it has open question about mining threads